lib/instrumentation/nr-winston-transport.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+const TransportStream = require('winston-transport')
+const logger = require('../logger').child({ compoonent: 'nr-winston-transport' })
+const { truncate } = require('../util/application-logging')
+
+/**
+ * Transport used to prepare a log line and add to the new relic agent
+ * log aggregator.
+ *
+ * Note*: This copies the log line so no other transports will get the
+ * mutated data.
+ */
+class NrTransport extends TransportStream {
+  constructor(opts = {}) {
+    opts.handleExceptions = true
```

Comment:
The comment you made in standup about the meaning of this `handleExceptions` option was useful. Can you add a one-line explanation of that this is in Winston, maybe just a link to the docs or similar?

Reply:
yea, good idea